frontends/web/src/routes/account/summary/summarybalance.tsx

```diff
@@ -48,34 +44,25 @@ type TProps = {
   totalBalance?: accountApi.TAccountTotalBalance;
   balances?: Balances;
   keystoreDisambiguatorName?: string;
+  lightningBalance: accountApi.IBalance | undefined;
```

Comment:
@thisconnect can this be simplified like this? same below in `TLightningProps`

```suggestion
  lightningBalance?: accountApi.IBalance;
```

Reply:
yes but with subtile difference. 

with this: `lightningBalance?: accountApi.IBalance;`  we tell TypeScript that the prop can be omitted.

with `lightningBalance: accountApi.IBalance | undefined;` the prop must be used but the value can be `undefined`